src/curve_fitting/model.clj

```diff
@@ -24,9 +24,15 @@
       (reduce + (map (gen [n] (* (nth coeffs n) (expt x n)))
                      (range degree))))))
 
+(define outlier?
+  (gen []
+    (flip 0.1)))
+
 (define add-noise-to-curve
   (gen [curve]
-    (gen [x] (gaussian (curve x) 0.1))))
+    (gen [x]
+      (gaussian (curve x)
+                (if (outlier?) 0.1 400)))))
```

Comment:
A few questions: 
- What is the third argument to `gaussian`?  The standard deviation?  The mean?
- Is there any significance to the `400` here or could this be any number that is far from `0.1`?
- I haven't fully wrapped my head around what the units of `0.1` and `400` are.  Any guidance about how to think about those would be appreciated.

Reply:
Gaussian only takes two arguments. The first argument is the mean (where the gaussian is "centered") and the second argument is the standard deviation (how wide the gaussian is).

Reply:
There's nothing magic about `0.1` and `400` here except that the former is small and the latter is pretty large relative to the size of the sketch. The intent was that if a given point is an inlier it should be "close" to the line. If a given point is an outlier it could be basically anywhere relative to the line.